tools/java/src/com/twitter/bazel/checkstyle/PythonCheckstyle.java

```diff
@@ -128,7 +128,9 @@ private static void runLinter(List<String> command) throws IOException {
 
     return Collections2.filter(spawnInfo.getInputFileList(),
         Predicates.and(
+          Predicates.or(
             Predicates.containsPattern(".*/src/.+\\.py[c]{0,1}$"),
+            Predicates.containsPattern("^heronpy/.+\\.py[c]{0,1}$")),
```

Comment:
indent could be improved for better reading, or put ')' in a new line.

Reply:
done